src/libs/ReportActionsUtils.ts

```diff
@@ -1348,6 +1350,34 @@ function isLinkedTransactionHeld(reportActionID: string, reportID: string): bool
     return TransactionUtils.isOnHoldByTransactionID(getLinkedTransactionID(reportActionID, reportID) ?? '-1');
 }
 
+function getAccountIDsFromMessage(message: string) {
```

Comment:
I don't think we really need this, because the format with the accountID mention is only created on the backend, at which point we also set a `mentionedAccountIDs` list on the `originalMessage`. So we can look at that instead.

Reply:
NAB: I see you're still using this. I don't think it's strictly needed, but I'm ok with leaving it in.